install/jakartaee/bin/ts.jte

```diff
@@ -1245,7 +1245,7 @@ command.testExecuteEjbEmbed=com.sun.ts.lib.harness.ExecTSTestCmd \
         ${JAVA_HOME}/bin/java \
         ${JVMOPTS_RUNTESTCOMMAND} \
         -Dcts.tmp=$harness.temp.directory \
-        -Djava.util.logging.config.file=${javaee.home}/domains/domain1/config/logging.properties \
```

Comment:
+1 for updating this ts.jte but not sure we need to update the other ones?

Reply:
this change might also be better made on the fly in the docker/run_jakartaeetck.sh, perhaps move the install/jakartaee/bin/client-logging.properties file into a glassfish file to make it move obvious it is part of the GlassFish porting kit and copy in if needed and update ts.jte file.

FYI I'm away from the computer myself for this coming week but other committers can help as needed.

Reply:
@gurunrao @alwin-joseph any thoughts on ^?

Reply:
It can be also removed from those other files, so it would use just JDK defaults, but then you cannot override it.
I don't know how those other files work and if they are even used, but they still always used server's logging.properties, which is wrong. Now they have just default file using console; the question is if they have the jar file on classpath. If not, we can still just change the content of the default-logging.properties to use just JDK classes.

Reply:
we can keep current modification for `client-logging.properties`. There will not be much benifit from moving same to `run_jakartaeetck.sh`